natives/windows-x86_64/src/main/java9/module-info.java

```diff
@@ -0,0 +1,18 @@
+/*
+ *    Copyright 2020-2021, Aayush Atharva
+ *
+ *    Brotli4j licenses this file to you under the
+ *    Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+module com.aayushatharva.brotli4j.windows.x86_64 {
+}
```

Comment:
A `requires com.aayushatharva.brotli4j` directive was omitted here, due to the project not having access to that module.